test/test_models.py

```diff
@@ -1,10 +1,19 @@
+from common_utils import TestCase, map_nested_tensor_object
 from collections import OrderedDict
 from itertools import product
 import torch
+import numpy as np
 from torchvision import models
 import unittest
 import traceback
+import random
 
+def set_rng_seed(seed):
+    torch.manual_seed(seed)
+    random.seed(seed)
+    if torch.cuda.is_available():
+        torch.cuda.set_rng_state(seed)
```

Comment:
I think this is not being called correctly, as `torch.cuda.set_rng_state` accepts a `state` object, which is a byte tensor.

I think we don't need to manually set the CUDA seed anymore, because it's done [by default when `torch.manual_seed` is called](https://github.com/pytorch/pytorch/blob/effcc398c447ce2aa6752354d64ef3d7ac5f004b/torch/random.py#L21-L34)